### ISSUE TITLE
Sprout Forms was recently updated

### ISSUE THREAD

**Issue by Takaya213** [barrelstrength/sprout-forms-countries#1]
The recent update to Sprout Forms on Craft 3 now gives the following error when using this input field and trying to browse already created forms with this field in it:

```
Error: Call to undefined method barrelstrength\sproutbase\app\fields\helpers\AddressHelper::getCountries() in /var/www/Applications/vendor/barrelstrength/sprout-forms-countries/src/integrations/sproutforms/fields/Countries.php:194
Stack trace:
#0 /var/www/Applications/vendor/barrelstrength/sprout-forms-countries/src/integrations/sproutforms/fields/Countries.php(42): barrelstrength\sproutformscountries\integrations\sproutforms\fields\Countries->getOptions()
#1 /var/www/Applications/vendor/yiisoft/yii2/base/BaseObject.php(109): barrelstrength\sproutformscountries\integrations\sproutforms\fields\Countries->init()
#2 /var/www/Applications/vendor/barrelstrength/sprout-forms-countries/src/SproutFormsCountries.php(47): yii\base\BaseObject->__construct()
#3 [internal function]: barrelstrength\sproutformscountries\SproutFormsCountries->barrelstrength\sproutformscountries\{closure}(Object(barrelstrength\sproutforms\events\RegisterFieldsEvent))
#4 /var/www/Applications/vendor/yiisoft/yii2/base/Event.php(310): call_user_func(Object(Closure), Object(barrelstrength\sproutforms\events\RegisterFieldsEvent))
#5 /var/www/Applications/vendor/yiisoft/yii2/base/Component.php(636): yii\base\Event::trigger('barrelstrength\\...', 'registerFieldsE...', Object(barrelstrength\sproutforms\events\RegisterFieldsEvent))
#6 /var/www/Applications/vendor/barrelstrength/sprout-forms/src/services/Fields.php(128): yii\base\Component->trigger('registerFieldsE...', Object(barrelstrength\sproutforms\events\RegisterFieldsEvent))
#7 /var/www/Applications/vendor/barrelstrength/sprout-forms/src/web/twig/variables/SproutFormsVariable.php(465): barrelstrength\sproutforms\services\Fields->getRegisteredFields()
#8 /var/www/Applications/vendor/twig/twig/lib/Twig/Extension/Core.php(1626): barrelstrength\sproutforms\web\twig\variables\SproutFormsVariable->getRegisteredFields()
#9 /var/www/Applications/vendor/craftcms/cms/src/helpers/Template.php(73): twig_get_attribute(Object(craft\web\twig\Environment), Object(Twig_Source), Object(barrelstrength\sproutforms\web\twig\variables\SproutFormsVariable), 'getRegisteredFi...', Array, 'method', false, false)
#10 /var/www/Applications/storage/runtime/compiled_templates/07/0796017e2384d80aa35cf522230dc5f95feb4338a44331fbdfd117f5e2fa334f.php(32): craft\helpers\Template::attribute(Object(craft\web\twig\Environment), Object(Twig_Source), Object(barrelstrength\sproutforms\web\twig\variables\SproutFormsVariable), 'getRegisteredFi...', Array, 'method')
#11 /var/www/Applications/vendor/twig/twig/lib/Twig/Template.php(386): __TwigTemplate_a09e758c4e3ae176f9a780214ba12c79fa4ca53413905f4761add5f5122bb120->doDisplay(Array, Array)
#12 /var/www/Applications/vendor/craftcms/cms/src/web/twig/Template.php(49): Twig_Template->displayWithErrorHandling(Array, Array)
#13 /var/www/Applications/vendor/twig/twig/lib/Twig/Template.php(363): craft\web\twig\Template->displayWithErrorHandling(Array, Array)
#14 /var/www/Applications/vendor/craftcms/cms/src/web/twig/Template.php(31): Twig_Template->display(Array, Array)
#15 /var/www/Applications/storage/runtime/compiled_templates/10/108ed5ba74d3f6843c7b33b21f1960688454c879847b6389b484d0a63289a714.php(271): craft\web\twig\Template->display(Array)
#16 /var/www/Applications/vendor/twig/twig/lib/Twig/Template.php(189): __TwigTemplate_7452e0ee81691739defd8638c3f640a4161e8aaab36eba98a4f78972a797c9c2->block_details(Array, Array)
#17 /var/www/Applications/vendor/twig/twig/lib/Twig/Template.php(255): Twig_Template->displayBlock('details', Array, Array, true)
#18 /var/www/Applications/storage/runtime/compiled_templates/66/6640124ed672ef6d5d200f8888206140780ae0fef3e1313c6fe5b85bea4cca61.php(82): Twig_Template->renderBlock('details', Array, Array)
#19 /var/www/Applications/vendor/twig/twig/lib/Twig/Template.php(386): __TwigTemplate_654807646b6e24797e0e9a4e217723fd211a43d60f1f784995cb5cf58fb90193->doDisplay(Array, Array)
#20 /var/www/Applications/vendor/craftcms/cms/src/web/twig/Template.php(49): Twig_Template->displayWithErrorHandling(Array, Array)
#21 /var/www/Applications/vendor/twig/twig/lib/Twig/Template.php(363): craft\web\twig\Template->displayWithErrorHandling(Array, Array)
#22 /var/www/Applications/vendor/craftcms/cms/src/web/twig/Template.php(31): Twig_Template->display(Array, Array)
#23 /var/www/Applications/storage/runtime/compiled_templates/91/919f48dddddf54ee5c7adfd76c14dbed61b0278b1b8bea2c2082bf8fce19a0da.php(34): craft\web\twig\Template->display(Array, Array)
#24 /var/www/Applications/vendor/twig/twig/lib/Twig/Template.php(386): __TwigTemplate_b4c1279bac4091c39c31c90392df6308f2b4ad643e21fbaa1bad8948e9af6961->doDisplay(Array, Array)
#25 /var/www/Applications/vendor/craftcms/cms/src/web/twig/Template.php(49): Twig_Template->displayWithErrorHandling(Array, Array)
#26 /var/www/Applications/vendor/twig/twig/lib/Twig/Template.php(363): craft\web\twig\Template->displayWithErrorHandling(Array, Array)
#27 /var/www/Applications/vendor/craftcms/cms/src/web/twig/Template.php(31): Twig_Template->display(Array, Array)
#28 /var/www/Applications/storage/runtime/compiled_templates/10/108ed5ba74d3f6843c7b33b21f1960688454c879847b6389b484d0a63289a714.php(118): craft\web\twig\Template->display(Array, Array)
#29 /var/www/Applications/vendor/twig/twig/lib/Twig/Template.php(386): __TwigTemplate_7452e0ee81691739defd8638c3f640a4161e8aaab36eba98a4f78972a797c9c2->doDisplay(Array, Array)
#30 /var/www/Applications/vendor/craftcms/cms/src/web/twig/Template.php(49): Twig_Template->displayWithErrorHandling(Array, Array)
#31 /var/www/Applications/vendor/twig/twig/lib/Twig/Template.php(363): craft\web\twig\Template->displayWithErrorHandling(Array, Array)
#32 /var/www/Applications/vendor/craftcms/cms/src/web/twig/Template.php(31): Twig_Template->display(Array, Array)
#33 /var/www/Applications/vendor/twig/twig/lib/Twig/Template.php(371): craft\web\twig\Template->display(Array)
#34 /var/www/Applications/vendor/twig/twig/lib/Twig/Environment.php(289): Twig_Template->render(Array)
#35 /var/www/Applications/vendor/craftcms/cms/src/web/View.php(337): Twig_Environment->render('sprout-forms/fo...', Array)
#36 /var/www/Applications/vendor/craftcms/cms/src/web/View.php(384): craft\web\View->renderTemplate('sprout-forms/fo...', Array)
#37 /var/www/Applications/vendor/craftcms/cms/src/web/Controller.php(161): craft\web\View->renderPageTemplate('sprout-forms/fo...', Array)
#38 /var/www/Applications/vendor/barrelstrength/sprout-forms/src/controllers/FormsController.php(195): craft\web\Controller->renderTemplate('sprout-forms/fo...', Array)
#39 [internal function]: barrelstrength\sproutforms\controllers\FormsController->actionEditFormTemplate(69, Object(barrelstrength\sproutforms\elements\Form))
#40 /var/www/Applications/vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array(Array, Array)
#41 /var/www/Applications/vendor/yiisoft/yii2/base/Controller.php(157): yii\base\InlineAction->runWithParams(Array)
#42 /var/www/Applications/vendor/craftcms/cms/src/web/Controller.php(109): yii\base\Controller->runAction('edit-form-templ...', Array)
#43 /var/www/Applications/vendor/yiisoft/yii2/base/Module.php(528): craft\web\Controller->runAction('edit-form-templ...', Array)
#44 /var/www/Applications/vendor/craftcms/cms/src/web/Application.php(297): yii\base\Module->runAction('sprout-forms/fo...', Array)
#45 /var/www/Applications/vendor/yiisoft/yii2/web/Application.php(103): craft\web\Application->runAction('sprout-forms/fo...', Array)
#46 /var/www/Applications/vendor/craftcms/cms/src/web/Application.php(286): yii\web\Application->handleRequest(Object(craft\web\Request))
#47 /var/www/Applications/vendor/yiisoft/yii2/base/Application.php(386): craft\web\Application->handleRequest(Object(craft\web\Request))
#48 /var/www/Applications/web/index.php(21): yii\base\Application->run()
#49 {main}
```

**Comment by andrelopez** [comment]
Hi @Takaya213 

This issue is fixed on v1.0.1, please also update to the latest Sprout Forms release (v3.0.0-beta.41)

Best,
-Andre L

**Comment by Takaya213** [comment]
Hi @andrelopez 

Thank you so much for fixing the issue! Appreciate it! :)
